packages/terra-brand-footer/src/BrandFooter.jsx

```diff
@@ -136,11 +136,13 @@ const BrandFooter = ({
               )
             }
             <ul className={cx('menu')}>
-              {linkGroup.links && linkGroup.links.map((link) => {
+              {linkGroup.links && linkGroup.links.map((link, index) => {
                 const spreadTarget = link.target !== undefined ? { target: link.target } : {};
+                const separator = (!isVertical && index >= 1) ? <span className={cx('separator')} aria-hidden>/</span> : '';
                 return (
                   <li className={cx('list-item')} key={link.text + link.href}>
                     <a className={cx('link')} href={link.href} {...spreadTarget}>
+                      {separator}
```

Comment:
we may have to reconsider where to render the separator within html markup - it's receiving the link's focus style. perhaps outside the list item?
![screen shot 2018-11-13 at 3 33 31 pm](https://user-images.githubusercontent.com/14063284/48444405-7ffa0f00-e759-11e8-9eb3-99ba219a6ae1.png)

Reply:
What browser are you seeing this in?

Reply:
I'm also seeing this, in Chrome. Moving it outside the item appears to fix it, so I'll push a change.